### ISSUE TITLE
Question: How to disable node simulation on nodeHover event?

### ISSUE THREAD

**Issue by attilamester** [vasturiano/react-force-graph#415]
Hi, 

I want to implement this awesome [example](https://vasturiano.github.io/react-force-graph/example/highlight/), but in my case, whenever I update the state, re-render occurs (to achieve the drawing of the on-hover-ring), and thus nodes are simulated.
I am using class components, I saw that in the above example you are using a `useMemo` effect, but still do not understand how does that stop the node simulation in your case. 
Any help is greatly appreciated, many thanks!

**Comment by attilamester** [comment]
I would like to mention some additional info: 
- the problem of nodes being simulated  seems to persist on any state update (eg. render node labels according to a flag in the state using `nodeCanvasObject`)
- the state is stored in redux (maybe that would be the cause of re-simulation?)
My question is: how can I update the state without the nodes being simulated?
Many thanks!

**Comment by attilamester** [comment]
Found out the root problem, sharing the "solution" here. The force-graph-component inits a simulation of the nodes if its `graphData` prop changes, even though the content is constant.
I used it in the following way: 
```
graphData={{nodes: this.state.nodes, links: this.state.links}}
``` 
This way, at each render, a new object reference was created, and simulation of nodes started.

PS. Kudos for the amazing library!
